lib/ansible/modules/cloud/dimensiondata/dimensiondata_network.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Dimension Data
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#   - Aimon Bustardo <aimon.bustardo@dimensiondata.com>
+#   - Bert Diwa      <Lamberto.Diwa@dimensiondata.com>
+#   - Adam Friedman  <tintoy@tintoy.io>
+#
+from ansible.module_utils.basic import *
+from ansible.module_utils.dimensiondata import *
+from ansible.module_utils.pycompat24 import get_exception
+try:
+    from libcloud.compute.types import Provider
+    from libcloud.compute.providers import get_driver
+    from libcloud.compute.base import NodeLocation
+    import libcloud.security
+    HAS_LIBCLOUD = True
+except:
+    HAS_LIBCLOUD = False
```

Comment:
It's preferred to move all of the imports below the DOCUMENTATION, EXAMPLES, and RETURN strings.  Those are data used externally to the module so they never have to be referred to when debugging the code.

If possible, change the wildcard imports (Like from ansible.module_utils.basic import *) into imports of specific things (like from ansible.module_utils.basic import AnsibleModule).